cmd/top-disk-spinner.go

```diff
@@ -40,7 +39,6 @@ type topDiskUI struct {
 
 	disksInfo map[string]madmin.Disk
 
-	mu         sync.Mutex
```

Comment:
@harshavardhana it is not clear in the bubbletea doc if View() and Update() can be called concurrently, that's why I added the mutex, LGTM otherwise

Reply:
View() is never called concurrent to Update() 

```
	// Update is called when a message is received. Use it to inspect messages
	// and, in response, update the model and/or send a command.
	Update(Msg) (Model, Cmd)

	// View renders the program's UI, which is just a string. The view is
	// rendered after every Update.
	View() string
```